app/views/restaurants/index.html.erb

```diff
@@ -1,14 +1,19 @@
 <h1>Welcome to our site!</h1>
 
 <h2>What do you feel like tonight?</h2>
-<input type='string' placeholder='eg. Thai, Italian, Burger'/>
-<input type='submit' value='Search' class='search-btn' />
 
-<div class='restaurants'>
-  <h2><%= 'Restaurants:' if @restaurants.any? %></h2>
-  <ul>
-    <% @restaurants.each do |restaurant| %>
-      <%= link_to restaurant.name, restaurant %>
-    <% end %>
-  </ul>
-</div>
+<%= form_tag(search_path) do %>
+  <%= text_field_tag :assortment, nil, placeholder: 'eg. Thai, American, Asian' %>
+  <%= submit_tag 'Search', class: 'search-btn' %>
+<% end %>
+
+<% if @results %>
+  <div class='restaurants'>
+    <h2><%= "Results for #{@assortment}" if @results.any? %></h2>
```

Comment:
This should be tested as a sad path if not results are available.

Reply:
We're adding that now